doc/userguide/src/CreatingTestData/AdvancedFeatures.rst

```diff
@@ -371,9 +371,9 @@ lower limit, upper limit and step.
 .. table:: For in range examples
    :class: example
 
-   ================  ===============  ===========  =========  ===========  ========  =======
-      Test Case          Action        Argument     Argument    Arg          Arg       Arg
-   ================  ===============  ===========  =========  ===========  ========  =======
+   ================  ===============  ===========  =========  ===========  ========  ========
+      Test Case          Action        Argument     Argument    Argument   Argument  Argument
+   ================  ===============  ===========  =========  ===========  ========  ========
```

Comment:
User Guide actually uses standard reStructuredText, not Sphinx. As I already commented elsewhere, the problem you had was not having an empty row after the table declaration.